src/main/java/io/rpg/viewmodel/LocationViewModel.java

```diff
@@ -45,4 +47,8 @@ public void initialize(URL location, ResourceBundle resources) {
       mapImageView.setFitHeight(newImg.getHeight());
     });
   }
+
+  public void addChild(ImageView child){
+    parent.getChildren().add(child);
```

Comment:
tak jak jest obecnie obrazki są rysowane obok tła, zamiast na nim. zmiana parent na contentPane chyba załatwia sprawę
```suggestion
    contentPane.getChildren().add(child);
```